CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 
 # [unreleased]
-
 ## New features
 
+- New syntax for memory load/store (uXX)[aligned? x e] become [aligned? :uXX x e],
+  the syntax  (uXX)[aligned? x e] is deprecated.
+  New syntax for array load/store t DOT? [aligned? uXX e] become t DOT? [aligned? :uXX e],
+  the syntax t DOT? [aligned? uXX e] is deprecated.
+  The script compiler/script/replace.sh allows to perform the change automatically.
+
+  New syntax for infix and prefix operators, the type of the arguments can be provided
+  using :uXX, like in e1 +:u32 e2.
```

Comment:
This line should also be markdownified. Maybe also highlights the fact that the current syntax in this case is not deprecated?